dbt_semantic_interfaces/validations/primary_entity.py

```diff
@@ -0,0 +1,102 @@
+import logging
+from typing import Generic, List, Sequence
+
+from dbt_semantic_interfaces.implementations.semantic_manifest import (
+    PydanticSemanticManifest,
+)
+from dbt_semantic_interfaces.protocols import Entity, SemanticManifestT, SemanticModel
+from dbt_semantic_interfaces.references import SemanticModelReference
+from dbt_semantic_interfaces.type_enums import EntityType
+from dbt_semantic_interfaces.validations.validator_helpers import (
+    FileContext,
+    SemanticManifestValidationRule,
+    SemanticModelContext,
+    ValidationError,
+    ValidationIssue,
+    validate_safely,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class PrimaryEntityRule(SemanticManifestValidationRule[SemanticManifestT], Generic[SemanticManifestT]):
```

Comment:
We should get this added to the [DEFAULT_RULES](https://github.com/dbt-labs/dbt-semantic-interfaces/blob/9cdf54593aa892fd969145bd53d7b2c891a6b62f/dbt_semantic_interfaces/validations/semantic_manifest_validator.py#L59) list if we want to ensure it's regularly enforced

Reply:
Added.